src/Core/Domain/Product/Exception/CannotUpdateProductException.php

```diff
@@ -125,4 +125,9 @@ class CannotUpdateProductException extends ProductException
      * When type update fails
      */
     public const FAILED_SHOP_COPY = 170;
+
+    /**
+     * When type update fails
```

Comment:
```suggestion
     * When product duplicaition fails
```
Altoguh If It were up to me, I would get rid of all those comments, as they don't add any value, they are basicaly same as constants name :/ (I would either have detailed comments which tells more than the constant name, or don't have them at all)

Reply:
I agree they don't bring that much value

Reply:
Typo Here